example/react-json-schema-form.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: 'ReadMe-Icons';
+  src:  url('https://i.icomoon.io/public/69485c64e8/ReadMeIcons/ReadMe-Icons.eot?5r4jo6');
+  src:  url('https://i.icomoon.io/public/69485c64e8/ReadMeIcons/ReadMe-Icons.eot?5r4jo6#iefix') format('embedded-opentype'),
+    url('https://i.icomoon.io/public/69485c64e8/ReadMeIcons/ReadMe-Icons.woff2?5r4jo6') format('woff2'),
+    url('https://i.icomoon.io/public/69485c64e8/ReadMeIcons/ReadMe-Icons.ttf?5r4jo6') format('truetype'),
+    url('https://i.icomoon.io/public/69485c64e8/ReadMeIcons/ReadMe-Icons.woff?5r4jo6') format('woff'),
+    url('https://i.icomoon.io/public/69485c64e8/ReadMeIcons/ReadMe-Icons.svg?5r4jo6#ReadMe-Icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'ReadMe-Icons' !important;
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Enable Ligatures ================ */
+  letter-spacing: 0;
+  -webkit-font-feature-settings: "liga";
+  -moz-font-feature-settings: "liga=1";
+  -moz-font-feature-settings: "liga";
+  -ms-font-feature-settings: "liga" 1;
+  font-feature-settings: "liga";
+  -webkit-font-variant-ligatures: discretionary-ligatures;
+  font-variant-ligatures: discretionary-ligatures;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+
+i.glyphicon {
+  display: none;
+}
+.btn-add::after {
+  font-family: 'ReadMe-Icons';
+  content: "\e975";
+  color: #028aec;
+}
+.btn.btn-default {
+  border: none;
+}
+.array-item-move-up::after {
```

Comment:
The styling on the following lines until line 57 never get reflected but I'm sure at some point we will them to work. The buttons still take up space and are clickable .